docs/integrations/README.md

```diff
@@ -138,7 +138,7 @@ For more information about the grading system, see [Product Release Stages](http
 | [Recurly](sources/recurly.md)                                                               | Alpha                | Yes                 |
 | [Redshift](sources/redshift.md)                                                             | Alpha                | Yes                 |
 | [Retently](sources/retently.md)                                                             | Alpha                | Yes                 |
-| [S3](sources/s3.md)                                                                         | Generally Available  | Yes                 |
+| [S3](sources/s3.md)                                                                         | Generally Available  | No                  |
```

Comment:
Hey @TBernstein4, the S3 source was shipped to GA in July. Can you point me to the source of the status change?